tests/PhpSpreadsheetTests/CommentTest.php

```diff
@@ -83,4 +84,12 @@ public function testSetText(): void
         $comment->setText($test);
         self::assertEquals('This is a test comment', (string) $comment);
     }
+    
+    public function testRemoveComment(): void {
+        $spreadsheet = new Spreadsheet();
+        $sheet = $spreadsheet->getActiveSheet();
+        $sheet->getComment('A2')->getText()->createText('Comment to delete');
+        $sheet->removeComment('A2');
```

Comment:
Probably also worth having an assert to ensure that `A2` has a comment after the create so that you can be sure that the `removeComment()` is removing something that was there